.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Upgrade to OCamlformat 0.26.0
+e2488a04be2d19e5c826914028104c480a744e60
```

Comment:
This file has helped me in the past. It's used with `git blame --ignore-revs-file=.git-blame-ignore-revs`.
Is this a welcome addition ?

Reply:
I've never seen this before but seems useful (and [automatically supported by github](https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/))!